app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar">
+  <ul>
+    <li>
+      <% if user_signed_in? %>
+        <%= link_to "Your Foods", foods_path %>
+      <% else %>
+        <%= link_to "Your Foods", new_user_registration_path %>
+      <% end %>
+
+    </li>
+    <li>
+      <% if user_signed_in? %>
+        <%= link_to "Your Recipes", recipes_path %> 
+      <% else %>
+        <%= link_to "Your Recipes", new_user_registration_path %>
+      <% end %>
+    </li>
+    <li>
+        <%= link_to "Recipes", public_recipes_path %>
+    </li>
+    <li>
```

Comment:
- [ ] It will be much more descriptive to name this nav member `Public Recipes` instead of `Recipes`.
- [ ] Kindly add the Generating Shopping List Page to be part of the nav items as well. 🙏 

<img width="607" alt="Untitled" src="https://github.com/agneta1991/recipe_app/assets/103386183/2bf0aa5d-bdbe-4908-b1c5-7a9b32c9c1da">